README.md

```diff
@@ -3,42 +3,49 @@
+OpenDP is a modular library of statistical algorithms that adhere to the definition of [differential privacy](https://en.wikipedia.org/wiki/Differential_privacy). It can be used to build applications of privacy-preserving computations, using a number of different models of privacy. OpenDP is implemented in Rust, with bindings for easy use from Python.
 
-## *WARNING*
-Please note that this library is still under development and not ready for production use. An important element of the OpenDP project is a formal vetting process that all library components must undergo, to verify their privacy characteristics. The code in this repository hasn't yet undergone that vetting. In addition to privacy concerns, it also likely has programming bugs.
+The architecture of OpenDP is based on a conceptual framework for expressing privacy-aware computations. This framework is described in the paper,
+[A Programming Framework for OpenDP](https://projects.iq.harvard.edu/files/opendp/files/opendp_programming_framework_11may2020_1_01.pdf).
 
-Feel free to explore the code, but *please do not yet rely on it for any privacy-sensitive applications*.
+OpenDP (the library) is part of the larger [OpenDP Project](https://opendp.org), a community effort to build trustworthy, open-source software tools for analysis of private data.
 
-## Overview
-OpenDP Core is a library of differential privacy algorithms for performing statistical analysis of private data. It consists of two conceptual layers:
+## Status
 
-* A theoretical framework for expressing privacy-aware operations. This framework is described in the paper,
-[A Programming Framework for OpenDP](https://projects.iq.harvard.edu/files/opendp/files/opendp_programming_framework_11may2020_1_01.pdf).
+OpenDP is under development, and we aim to [release new versions](https://github.com/opendp/opendp/releases) frequently. It's a work in progress, but we feel that OpenDP already can be used to build some useful applications. We also hope that it may be a vehicle for exploring new ideas in privacy. We welcome you to try it, and look forward to feedback on the usability of the library! However, please be aware of the following limitations.
 
-* A set of algorithmic components implemented within the conceptual framework. These components can be used "out of the box" to build applications for handling private data.
+### WARNING
 
-## Implementation
-OpenDP Core is packaged as a library that can be incorporated into applications. Public APIs to the library are provided in Python and Rust, with more language bindings possible based on interest. The underlying logic of the library is implemented in native Rust.
+OpenDP is not yet recommended for production deployments. It has known privacy concerns, particularly around floating point handling and side channel attacks. We expect to address these issues, but until then, *please don't use OpenDP for any privacy-critical applications*.
```

Comment:
If it makes sense, perhaps we should create issues for these concerns and link to them.